sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py

```diff
@@ -75,7 +71,7 @@ class CloudEvent(EventMixin):   #pylint:disable=too-many-instance-attributes
      unique for each distinct event.
     :type id: Optional[str]
     """
-    def __init__(self, source, type, **kwargs):
+    def __init__(self, source, type, **kwargs): # pylint: disable=redefined-builtin
```

Comment:
Why is that?

Reply:
`type` - unfortunately cant rename this either given cloud event spec